README.md

```diff
@@ -193,8 +193,8 @@ All rights reserved.
     <https://wiki.openssl.org>
     "OpenSSL Wiki"
 
-[OpenSSL 3.0 Wiki]:
-    <https://wiki.openssl.org/index.php/OpenSSL_3.0>
+[migration_guide(7ossl)]:
+    <https://www.openssl.org/docs/man3.0/man7/migration_guide.html>
     "OpenSSL 3.0 Wiki"
```

Comment:
You've kept `OpenSSL 3.0 Wiki` here. Also wouldn't it be better to just drop the link? The manual page is available locally if built. Or should the link be to master version?

Reply:
> You've kept `OpenSSL 3.0 Wiki` here.

Ups. Will be fixed.

> Also wouldn't it be better to just drop the link? The manual page is available locally if built.

The link is intended and useful for visitors of our landing page https://github.com/openssl/openssl which skim over the README.

>  Or should the link be to master version?

Actually, on master it should point to 

https://www.openssl.org/docs/manmaster/man7/migration_guide.html

and the cherry-picked commit on the 3.0 stable branch should point to 

https://www.openssl.org/docs/man3.0/man7/migration_guide.html

if you don't mind that it's changed while backporting.

Reply:
> if you don't mind that I‘ll change it while backporting.

Better prepare separate PR for master and 3.0 branches.

Reply:
Done, see #17849.